integrations/aws/main.py

```diff
@@ -33,21 +37,51 @@
 )
 
 
+async def _handle_global_resource_resync(
+    kind: str,
+    credentials: AwsCredentials,
+    current_session: aioboto3.Session,
+    handle_exceptions: bool = True,
+) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    try:
+        async for batch in resync_cloudcontrol(kind, current_session):
+            yield batch
+    except Exception as e:
+        if is_access_denied_exception(e):
+            if handle_exceptions:
+                logger.info(f"Trying to resync {kind} in all regions until success")
+                async for async_session in credentials.create_session_for_each_region():
+                    session = await async_session
+                    try:
+                        async for batch in resync_cloudcontrol(kind, session):
+                            yield batch
+                        break
+                    except Exception as e:
+                        continue
```

Comment:
also we shouldn't silent fail, only when access denied or not found and also then to log it